db/migrate/20231021100348_change_annotations_status_default_in_media_and_lectures.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeAnnotationsStatusDefaultInMediaAndLectures < ActiveRecord::Migration[7.0]
+  def change
+    change_column_default :media, :annotations_status, 0
+    change_column_default :lectures, :annotations_status, -1
```

Comment:
What does it mean if the annotation status is `0` or `-1`? The only case I see the value `-1` is in the `close_comments` method in the lectures controller. How is `0` different from `-1`?

Reply:
If we only needed a binary state for "on/off", then a boolean would suffice, right? That's why I assume that `-1` has a special meaning which I did not find out yet.